stage1/appexec/appexec.c

```diff
@@ -256,6 +260,16 @@ static void parse_gids(const char *str, size_t *n_gids_p, gid_t **gids_p)
 
 int main(int argc, char *argv[])
 {
+	char *term = getenv("TERM"); /* usefull to keep during entering. */
+	int c;
+
+	while ((c = getopt(argc, argv, "e")) != -1)
+		switch (c) {
+			case 'e':
+				setenv("TERM", term, 1);
```

Comment:
`setenv("TERM", term, 1);` is called immediately after the `char *term = getenv("TERM");` here, and before the `load_env() / clearenv()`. It seems to be a no-op, and not what we want.